component/loki/source/cloudflare/cloudflare.go

```diff
@@ -0,0 +1,144 @@
+package cloudflare
```

Comment:
TODO: Check if there's a good way to test the component itself without making it too messy with the mocks.